src/components/Content/News/News.tsx

```diff
@@ -1,33 +1,64 @@
-import React, { createContext, useState } from 'react';
-import { NewsList } from './NewsList/NewsList';
-import { NewsItemDetails } from './NewsDetails/NewsDetails';
+import React from 'react';
+import {NewsList} from './NewsList/NewsList';
 import styled from 'styled-components';
-import { useAppSelector } from '../../../redux/hooks/hooks';
-import { selectSidebarNewsExpanded } from '../../../redux/features/sidebarNewsSlice';
+import {useAppDispatch, useAppSelector} from '../../../redux/hooks/hooks';
+import {
+    fetchNews,
+    selectCurrentNews, selectNewsFetchError,
+    selectNewsIsPending,
+    selectNewsPageSize,
+    selectNextNewsPage, setPage, setPageSize
+} from '../../../redux/features/newsSlice';
+import {useAPIFetch} from '../../../redux/hooks/useAPIFetch';
+import {Preloader} from '../../common/Preloader/Preloader';
+import {APIError} from '../../service/APIError/APIError';
 
 type NewsProps = {};
 
-const StyledNewsWrapper = styled.div<{ $sidebarNewsExpanded: boolean }>`
-    display: grid;
-    grid-template-columns: ${(props) =>
-        props.$sidebarNewsExpanded ? '0fr 1fr' : '1fr 0'};
-    transition: grid-template-columns 500ms;
-    overflow: hidden;
-
-    ${(props) => (props.$sidebarNewsExpanded ? 'overflow-y: hidden' : '')};
-
+const StyledNewsWrapper = styled.div`
     @media (max-width: 600px) {
         font-size: 10px;
     }
 `;
 
 export const News: React.FC<NewsProps> = ({}) => {
-    const sidebarNewsExpanded = useAppSelector(selectSidebarNewsExpanded);
+    const dispatch = useAppDispatch();
+
+    dispatch(setPageSize(50));
+    dispatch(setPage(0));

```

Comment:
It's very very not recommended to dispatch actions right when the component is trying to mount. It causes at least one more render and it might cause problems in future...What do you think if we keep that in redux initial state?